pkg/virt-handler/vm.go

```diff
@@ -2967,3 +2966,19 @@ func (d *VirtualMachineController) reportTargetTopologyForMigratingVMI(vmi *v1.V
 	vmi.Status.MigrationState.TargetNodeTopology = string(topology)
 	return nil
 }
+
+func (d *VirtualMachineController) handleMigrationAbort(vmi *v1.VirtualMachineInstance, client cmdclient.LauncherClient) error {
+	if vmi.Status.MigrationState.AbortStatus == v1.MigrationAbortInProgress {
+		return nil
+	}
+
+	err := client.CancelVirtualMachineMigration(vmi)
+	if err != nil && err.Error() != migrations.CancelMigrationFailedVmiNotMigratingErr {
+		// If migration did not even start there is no need to cancel it
+		log.Log.Object(vmi).Infof("skipping on migration cancellation since vmi is not migrating")
+		return err
+	}
```

Comment:
is this if statement structured correctly? It looks and reads a bit odd.

Reply:
You're right, sorry for that.
What about `"skipping migration cancellation since vmi is not migrating"`? (removed redundant "on")
Or maybe `"skipping migration cancellation since no migration ongoing for vmi %s"`?